ui-v2/app/controllers/dc/acls/edit.js

```diff
@@ -1,30 +1,31 @@
 import Controller from '@ember/controller';
-import { set } from '@ember/object';
-import Changeset from 'ember-changeset';
-import validations from 'consul-ui/validations/acl';
-import lookupValidator from 'ember-changeset-validations';
+import { inject as service } from '@ember/service';
+import { get } from '@ember/object';
 
 export default Controller.extend({
+  builder: service('form'),
+  dom: service('dom'),
+  init: function() {
+    this._super(...arguments);
+    this.form = get(this, 'builder').form('acl');
+  },
   setProperties: function(model) {
-    this.changeset = new Changeset(model.item, lookupValidator(validations), validations);
-    this._super({
-      ...model,
-      ...{
-        item: this.changeset,
-      },
-    });
+    // essentially this replaces the data with changesets
+    this._super(
+      Object.keys(model).reduce((prev, key, i) => {
+        switch (key) {
+          case 'item':
+            prev[key] = this.form.setData(prev[key]).getData();
+            break;
+        }
+        return prev;
+      }, model)
+    );
```

Comment:
I tried simplifying this a little, by `Object.assign`ing instead. Strangely doing that in the `token` form broke my tests, see further down, I'll comment in where it broke.

Reply:
Ah I can't theres no change there 😂 , link instead:

https://github.com/hashicorp/consul/blob/b153f9be39d30a772a130b39560a01b9e29ad3dd/ui-v2/app/controllers/dc/acls/tokens/edit.js#L12-L30

So changing that ^ to use `Object.assign` broke the tests, so I left things as they are for now.

I might look into it a little deeper, but I want to start feature work again on Monday so I don't want to get in a rabbit hole refactoring too much.